mLRS/Common/hal/esp/rx-hal-generic-2400-pa-td-esp32.h

```diff
@@ -0,0 +1,194 @@
+//*******************************************************
+// Copyright (c) MLRS project
+// GPL3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+// OlliW @ www.olliw.eu
+//*******************************************************
+// hal
+//********************************************************
+
+//-------------------------------------------------------
+// GENERIC 2400 True Diversity PA RX
+//-------------------------------------------------------
+
+#define DEVICE_HAS_SINGLE_LED
+//#define DEVICE_HAS_SERIAL_OR_DEBUG
+#define DEVICE_HAS_NO_DEBUG
+
+//-- Timers, Timing, EEPROM, and such stuff
+
+#define DELAY_USE_DWT
+
+#define SYSTICK_TIMESTEP          1000
+#define SYSTICK_DELAY_MS(x)       (uint16_t)(((uint32_t)(x)*(uint32_t)1000)/SYSTICK_TIMESTEP)
+
+#define EE_START_PAGE             0 // 128 kB flash, 2 kB page
+
+#define MICROS_TIMx               TIM15
+
+//-- UARTS
+// UARTB = serial port
+// UART = output port, SBus or whatever
+// UARTC = debug port
+
+#define UARTB_USE_SERIAL
+#define UARTB_BAUD                RX_SERIAL_BAUDRATE
+#define UARTB_USE_TX
+#define UARTB_TXBUFSIZE           RX_SERIAL_TXBUFSIZE // 1024 // 512
+#define UARTB_USE_TX_ISR
+#define UARTB_USE_RX
+#define UARTB_RXBUFSIZE           RX_SERIAL_RXBUFSIZE // 1024 // 512
+
+#define UARTC_USE_SERIAL
+#define UARTC_BAUD                  115200
+
+//#define SWUART_USE_TIM15 // debug
+#define SWUART_TX_IO              10
+#define SWUART_BAUD               57600
+#define SWUART_USE_TX
+#define SWUART_TXBUFSIZE          512
+
+//-- SX1: SX12xx & SPI
+
+#define MISO                      33
+#define MOSI                      32
+#define SCK                       25
+#define SPI_FREQUENCY             16000000L
+
+#define SX_BUSY                   36
+#define SX_DIO1                   37
+#define SX_NSS                    27
+#define SX_RESET                  26
+
+#define SX_RX_EN                  10
+#define SX_TX_EN                  14
+
+#define SX_BUSY_2                 39
```

Comment:
mLRS convention is SX2_xxxx, not SX_xxxx_2

Reply:
Addressed